evap/student/templates/student_vote.html

```diff
@@ -168,6 +188,21 @@ <h3 class="mb-3">{{ course.name }} ({{ course.semester.name }})</h3>
                 return false;
             });
 
+
+            function markUnansweredAsNoAnswerFor(contributorId) {
+                rows = $("#vote-area-" + contributorId).find(".vote-row");
+                for (i = 0; i < rows.length; i++) {
+                    row = $(rows[i]);
+                    vote_inputs = row.find(".vote-inputs [type=radio]");
+
+                    if (!vote_inputs.parent().hasClass("active")) {
+                        row.find("[value=6]").prop('checked', true);
+                        row.find("[value=6]").parent().addClass("active");
+                        sisyphus.saveAllData();
```

Comment:
In case this method will be kept (it's called in 250 as far as i can see): Is this call really necessary once per row or can we just do it once after the loop completed?